framework/include/utils/PiecewiseByBlockLambdaFunctor.h

```diff
@@ -104,6 +107,12 @@ class PiecewiseByBlockLambdaFunctor : public Moose::FunctorBase<T>
   /// Functors that will evaluate elements at side quadrature points
   std::unordered_map<SubdomainID, ElemSideQpFn> _elem_side_qp_functor;
 
+  /// The name of this object
+  std::string _name;
```

Comment:
@GiudGiud don't we have `_name` in the `FunctorBase` now?

Reply:
yeah we do have names for functors up there now

Reply:
Deleted!